shopinvader/models/sale.py

```diff
@@ -209,6 +210,14 @@ class SaleOrderLine(models.Model):
         string="Shopinvader Variant",
     )
 
+    @api.multi
+    def shopinvader_recompute(self):
+        for line in self:
```

Comment:
@rousseldenis This method should be on sale.order.line ????!!! Moreover, you break batch re computation! You must call recompute on all the records at once.

Reply:
It's on sale order line (the models are mixed...)

Ok for recompute on recordset